Update Cloud Quickstart (#18950)
cockroachdb/docs: src/current/cockroachcloud/quickstart.md

---
title: Quickstart with CockroachDB

> summary: Learn how to use free trial credits to deploy a CockroachDB cluster on CockroachDB Standard (Preview), then use sample code to run your first queries.

Reviewer: ```suggestion
summary: Learn how to use free trial credits to deploy a CockroachDB cluster on CockroachDB Standard (Preview) and run your first queries.
```

toc: true
cloud: true
referral_id: docs_quickstart_free
docs_area: get_started
---

> This page shows you how to use free trial credit to deploy a CockroachDB cluster on CockroachDB {{ site.data.products.standard }} (Preview), and use sample code to run your first queries.

Reviewer 2: "free trial credit" should be plural

Author: SGTM. Thanks!

> This page shows you how to use free trial credits to deploy a CockroachDB cluster on CockroachDB {{ site.data.products.standard }} (Preview), and use sample code to run your first queries.

Reviewer: ```suggestion
This page shows you how to use free trial credits to deploy a CockroachDB cluster on CockroachDB {{ site.data.products.standard }} (Preview) and use sample code to run your first queries.
```
